hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/common/Storage.java

```diff
@@ -145,7 +145,7 @@ protected StorageInfo getStorageInfo() {
   abstract protected Properties getNodeProperties();
 
   /**
-   * Sets the Node properties specific to OM/SCM.
+   * Sets the Node properties specific to OM/SCM/DataNode.
```

Comment:
Can we add a unit test for the Datanode layout version startup validation like in TestScmStartupSlvLessThanMlv, TestOmStartupSlvLessThanMlv?

Reply:
yes